app/components/elements/PillButton.element.tsx

```diff
@@ -31,16 +31,16 @@ export const PillButton = styled.button<PillProps>`
   border-width: 1px;
   border-color: transparent;
   border-style: solid;
-  
-  padding: .25rem 15px .25rem;
-  margin: .25rem;
-  
-  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
-  
+
+  padding: 0.25rem 15px 0.25rem;
```

Comment:
you specified everything for padding except the left number, I think you have to have 1(every direction), 2(vertical and horizontal), or 4(each separate direction) specified numbers, not sure how just putting 3 works even if its 0 I would put it

Reply:
i think i do not know how to write code...

Reply:
wait how did you even find this i didn't touch it lmao

Reply:
but fun fact if you provide 3 then the last value will match the second one - fixed cause CSS magic is not clean code